src/classes/STG_PanelRDHealthCheck.cls

```diff
@@ -197,6 +199,27 @@ public inherited sharing class STG_PanelRDHealthCheck {
         }
     }
 
+    /**
+    * @description Validate that all RD Status are mapped to valid States
+    * @return void
+    */
+    @TestVisible
+    private void validateStatusMapping() {
+        List<SelectOption> picklistValues = 
+            UTIL_Describe.getSelectOptions(RD_SOBJECT_NAME, String.valueOf(npe03__Recurring_Donation__c.Status__c));
+
+        for (SelectOption option : picklistValues) {
+            if (!option.getDisabled() && String.isBlank(RD2_StatusMapper.getInstance().getState(option.getValue()))) {
+                result.addError(
```

Comment:
How does this error message look like, is there a space between two sentences? 
The test verifies that error contains `RD2_StatusMustBeMapped`, but does not check the complete message that includes `RD2_StatusMustBeMappedSolution`.

Reply:
Ignore the above, this is  Health Check error that is separate from the solution. The validation concats strings, but it is not done here.